apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -574,7 +577,8 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
       <div
         className={moduleStyles.inputsContainer}
         tabIndex={0}
-        onClick={() => handleStartOverClick(true)}
+        onClick={handleStartOverClick}
+        data-usingheader
```

Comment:
Since it's a boolean value, it's a sorta special case. We don't need to explicitly say

```
data-usingheader={true}
```

since it's the same thing. I know there's a linter setting that'll actually complain if you do that, too, but I don't think we have it enabled. So it's just handed in w/o value and implies true.